code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -27,7 +27,9 @@
 	/// Number of containers to be printed
 	var/printing_total
 	/// The amount of containers that can be printed in 1 cycle
-	var/printing_amount = 1
+	var/printing_amount = 50
```

Comment:
You can now make this a constant in both UI and backend, no need for a var

Reply:
Also a delay of 0.75 seconds and 0.25 seconds makes very little difference from a player perspective.

Consider. giving it a larger initial value and step down from there

Reply:
With how long it takes to print 50, there is roughly a 30 second difference between tier 1 and tier 3 as tested, which does provide incentive to upgrade.

Reply:
Made it into a constant.